client/src/components/DetailsApp.jsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import axios from 'axios';
+
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      data: {}
+    };
+  }
+
+  componentDidMount() {
+    // console.log('window.location', window.location);
+    // console.log('window.location.pathname:', window.location.pathname);
+    axios.get(`http://localhost:3001/details${window.location.pathname}`).then(res => {
```

Comment:
perhaps consider replacing references to 'http://localhost:3001' with a short circuited var in case your ports ever need to change. I don't think this is crucial, but maybe an optimization?

Reply:
Really really good question. I'm keeping it for right now since we'll be pointing to an IP address when we deploy to AWS from my understanding.